frontend/src/actions/WindowActions.js

```diff
@@ -530,6 +531,7 @@ export function createSearchWindow({
       isModal,
     })
   ).then(({ windowId, viewId }) => {
+    dispatch(setPendingAdvSearch(false));
```

Comment:
pls also `setPendingAdvSearch(false)` in case of a failure  (`finally`?).

Reply:
good point. Will change this